.github/workflows/cd_release.yml

```diff
@@ -131,13 +131,10 @@ jobs:
             system_type: ["win", ""]
             arch: amd64
             py_minors: '11'
-
-          # See issue #221: https://github.com/SINTEF/dlite/issues/221
-          # # 64-bit (Intel) macOS
-          # - os: macos-10.15
-          #   system_type: ["macosx", ""]
-          #   arch: x86_64
-          #   py_minors: 7,8,9,10
+          - os: windows-2019
+            system_type: ["win", ""]
+            arch: amd64
+            py_minors: '12'
```

Comment:
Can the windows releases be combined into one, similar to the linux ones?
`py_minors: 7,8,9,10,11,12`

Reply:
I don't know. I have just assumed that there is a reason why  they were separated initially. 

We should check it out, it just takes a lot of time to wait for the wheel to be build...